constraints/cuda.txt

```diff
@@ -1 +1 @@
---extra-index-url https://download.pytorch.org/whl/cu118
+--extra-index-url https://download.pytorch.org/whl/nightly/cu118
```

Comment:
Should we bump CUDA to the latest 12.1?

https://download.pytorch.org/whl/nightly/cu121

Reply:
12.1 does not support the CI gpus